docs/video/smoothie/recipe.md

```diff
@@ -38,13 +50,16 @@ Here's what each file is for:
 
 ## :material-blur-linear: frame blending { #frame-blending }
 
-Just like blur's "blur" config category, VEGAS' smart resampling and FFmpeg's `tmix` filter but much faster, this averages every frame with it's neighboring frames, creating trails and if done correctly looks like realistic motion-blur.
+`[frame blending]` is just like blur's `- blur` config category, VEGAS' smart resampling, and FFmpeg's `tmix` filter - but much faster. It combines frames with neighboring ones to decrease the frame rate but preserve smoothness. It's recommended to blend to 60FPS or less, because the majority of websites and monitors cap at 60.
```

Comment:
(atleast for blur/Smoothie frame blending) you don't necessarily have to decrease the frame rate (just set the output fps to input)